mappings/net/minecraft/command/argument/serialize/ArgumentSerializer.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2314 net/minecraft/command/arguments/serialize/ArgumentSerializer
+CLASS net/minecraft/class_2314 net/minecraft/command/argument/serialize/ArgumentSerializer
```

Comment:
Out of scope for this PR, but this should probably be `argument.serializer` instead of `argument.serialize`.